_posts/2016/2016-07-07-elastic-meetup.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: "Elastic Meetup at ePages"
+date: "2016-07-08 11:11:11"
+image: blog-header/elastic-meetup.jpg
```

Comment:
![elastic-meetup](https://cloud.githubusercontent.com/assets/10460958/16684534/d58ba5c6-4505-11e6-822d-e756d5fb1874.jpg)

Please use this size - prepared it for the blog.
Btw: where is this picture from? Pixabay? Pexels? Or is it your own?